etl/controller/misseddata.go

```diff
@@ -88,10 +88,11 @@ func (mdm *MissedDataManager) deleteExpired() {
 	defer mdm.mutex.Unlock()
 
 	for i, missed := range mdm.missedDataPool {
```

Comment:
May be its better to make here copy of `mdm.missedDataPool` and iterate over it? Its seems to be more clear